src/stateProviders/provideSavedQueries.js

```diff
@@ -15,33 +15,44 @@ export default provideState({
     removeDeletingId: update((state, id) => ({
       deletingIds: state.deletingIds.filter(dId => dId !== id),
     })),
-    getQueries: (effects, egoId) => {
+    getQueries: (effects, { egoId, api }) => {
       const jwt = localStorage.getItem('EGO_JWT');
       return effects
         .setLoading(true)
-        .then(() =>
-          fetch(urlJoin(shortUrlApi, 'user', egoId), {
-            method: 'GET',
-            headers: {
-              Authorization: `Bearer ${jwt}`,
-            },
-          }),
+        .then(
+          () =>
+            api
+              ? api({
+                  url: urlJoin(shortUrlApi, 'user', egoId),
+                  method: 'GET',
+                })
+              : fetch(urlJoin(shortUrlApi, 'user', egoId), {
```

Comment:
Is this one necessary too? There's only one instance of `getQueries` being called, passing in the `api`... maintaining the pattern seems like a good idea, rather than providing defaults here & there